cmd/ledgerutil/main.go

```diff
@@ -60,19 +62,24 @@ func main() {
 			}
 		}
 
-		count, outputDirPath, err := ledgerutil.Compare(*snapshotPath1, *snapshotPath2, *outputDir, *firstDiffs)
+		count, outputDirPath, err := ledgerutil.Compare(*snapshotPath1, *snapshotPath2, *outputDir, *firstDiffs, *namespace)
 		if err != nil {
 			fmt.Printf("%s%s\n", compareErrorMessage, err)
 			return
 		}
 
 		fmt.Print("\nSuccessfully compared snapshots. ")
-		if count == -1 {
+		if *namespace != "" {
+			fmt.Printf("The namespace \"%s\" was used as a filter. ", *namespace)
+		}
```

Comment:
Perhaps this should be moved before you call `ledgerutil.Compare`  above?  And maybe something like `Comparing namespace ...` and if not set `Comparing all namespaces` ?

Reply:
So some logging before the tool starts the comparison to let the user know it's running and under what parameters? Good suggestion. I think it might also be worth logging to the user the `firstDiffs` value as well.

Reply:
Small Go trick: You can use backticks so you dont have to escape quotes: `\`The namespace \"%s\" was used as a filter.\``